docs/data/material/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -156,34 +156,50 @@ import Button from '@mui/material/Button/Button.js';
 Future changes to the library's internal structure could break these paths. `babel-plugin-direct-import` allows for granular control over what gets imported, but it comes with the potential risk of relying on internal library paths. This may fail in future versions if the package is updated to use the `exports` field in `package.json`, which could block access to internal paths like this.
 :::
 
-If you are using Create React App, you will need to use a couple of projects that let you use `.babelrc` configuration, without ejecting.
+Material UI supports tree shaking out of the box when importing from specific paths (such as `@mui/material/Button`), so this configuration is optional and primarily useful if you want to enforce modular imports.
 
-`yarn add -D react-app-rewired customize-cra`
+Vite doesn't require extra Babel configuration by default because it uses esbuild for fast bundling and minification.
+But if you need to customize Babel (for example, to use babel-plugin-import), you can follow these steps:
```

Comment:
`babel-plugin-import` is not used below.

Reply:
it should be babel-plugin-transform-import